### PR TITLE
feat: creation of InterventionListView

### PR DESCRIPTION
**Pull request by vhu-axelor**
RM#76308

### CODE REVIEW

**Comment by vhu-axelor** [comment]
Waiting merge of ticket #76743

File: packages/apps/intervention/src/components/molecules/InterventionDetailCard/InterventionDetailCard.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+  const phone = useMemo(
+    () =>
+      intervention?.deliveredPartner?.mobilephone ||
+      intervention?.deliveredPartner?.fixedPhone,
+    [
+      intervention?.deliveredPartner?.fixedPhone,
+      intervention?.deliveredPartner?.mobilephone,
+    ],
+  );
```
Comment (lme-axelor) [inline packages/apps/intervention/src/components/molecules/InterventionDetailCard/InterventionDetailCard.tsx:62]: you should use the informations directly saved on the intervention object.

File: packages/apps/intervention/src/models/searchFields.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {SearchFields} from '@axelor/aos-mobile-core';
+
+export const intervention_searchFields: SearchFields = {
+  intervention_intervention: ['sequence'],
```
Comment (lme-axelor) [inline packages/apps/intervention/src/models/searchFields.ts:22]: Add partner fullname please

File: packages/apps/intervention/src/models/sortFields.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {SortFields} from '@axelor/aos-mobile-core';
+
+export const intervention_sortFields: SortFields = {
+  intervention_intervention: ['-planifStartDateTime'],
```
Comment (lme-axelor) [inline packages/apps/intervention/src/models/sortFields.ts:22]: order is inversed I think

File: packages/apps/intervention/src/screens/intervention/InterventionsHistoryScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {InterventionsListView} from '../../components';
+import {Intervention} from '../../types';
+
+const InterventionsHistoryScreen = ({}) => {
+  return <InterventionsListView statusList={[Intervention.status.Finished]} />;
```
Comment (lme-axelor) [inline packages/apps/intervention/src/screens/intervention/InterventionsHistoryScreen.tsx:24]: Missing screen component to remove space on top (same on planned interventions)

**Pull request merged**